construi/config.py

```diff
@@ -33,7 +33,8 @@ def for_target(self, target):
         construi = {
             'before': target_yml['before'] if 'before' in target_yml else [],
             'name': target,
-            'run': self.target_yml(target).get('run', [])
+            'run': self.target_yml(target).get('run', []),
+            'working_dir': self.working_dir
```

Comment:
We don't want to add the working_dir here. This is passed to the container create and sets the working directory inside the container.